selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/filewatcher/CheckFileWatcherExcludeFeatureTest.java

```diff
@@ -59,32 +59,26 @@ public void setUp() throws Exception {
         PROJECT_NAME,
         ProjectTemplates.MAVEN_SPRING);
     ide.open(testWorkspace);
-
     projectExplorer.waitProjectExplorer();
     projectExplorer.waitItem(PROJECT_NAME);
+    menu.runCommand(
```

Comment:
To make code clearer, could you, please, explain why we need to do these steps in comment?

Reply:
So we need do this because the fileWatcherIgnore file is hidden. This file is used for store list files/folders for filewatcher excluding. In the test we open this file for checking which files have been added to exclude.

Reply:
I see. Let's add comment into the code.

Reply:
done